packages/core-api/src/apis/implementations/UserSettingsApi/index.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useState, useEffect } from 'react';
+import { UserSettingsApi } from '../../definitions';
+
+export class UserSettings implements UserSettingsApi {
```

Comment:
Keep index small and put this in e.g. `UserSettings` instead imo. Maybe rename to something that describes this particular implementation a bit more, like `WebStorageUserSettings`